### PR TITLE
Making site mobile responsive

### PR DESCRIPTION
**Pull request by b0llu**
image
image
image
image

### CODE REVIEW

File: cart-page/cart.html [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+                            <button class="btn">Remove from Cart</button>
+                        </div>
+                    </div>
+                </div>
+                <div class="ammount-container stick">
+                    <div class="card-container card-shadow">
+                        <h2 class="card-subtitle bold">Price Detail</h2>
+                        <div class="filter-divider-line"></div>
+                        <h2 class="card-subtitle">Price: <span>90000Rs</span></h2>
+                        <h2 class="card-subtitle">Quantity: <span>3</span></h2>
```
Comment (ApurvaSawant11) [inline cart-page/cart.html:146]: Not related to development, but `Rs` should not be written after the amount. It should always be `Rs. 90000`. Or use `₹` before the amount

**Review by ApurvaSawant11 — APPROVED** [review]
Looks great! Loved your UI

**Review by Hamza-777 — APPROVED** [review]
Looks Good.

File: cart-page/cart.html (hunk 1 above)
Comment (b0llu) [reply cart-page/cart.html:146]: thanks for that apurva I will change it around

**Pull request merged**